lib/Graph/Graph.cpp

```diff
@@ -494,6 +494,16 @@ MaxPoolNode *Function::createMaxPool(llvm::StringRef name, NodeValue input,
   return addNode(new MaxPoolNode(name, OT, input, kernels, strides, pads));
 }
 
+MaxPoolNode *Function::createMaxPool(llvm::StringRef name, NodeValue input,
+                                     TypeRef outTy,
```

Comment:
Something to double check but I am not sure the interpreter would work properly with an outTy that have different scale and offset as the input type.

Reply:
We make sure that the output scale/offset is the same with the input.